chcemvediet/apps/inforequests/templates/inforequests/detail/action/main.html

```diff
@@ -14,8 +14,9 @@
 {% endcomment %}
 
 
-<div id="a{{ action.pk }}">
-  {% with print_id=idgenerator.next %}
+{% with action_id='a'|add:action.pk|stringformat:"s" %}
+<div id="{{ action_id }}">
+  {% with print_id=action_id %}
```

Comment:
Ked chces tlacit element, ktory uz ma ID, tak premennu `print_id` v principe nepotrebujes a mozes rovno v `controls.html` pouzit `action_id` ako print target.

A kedze ID elementu s akciou ma odvoditelnu hodnotu, tak ani premennu `action_id` nepotrebujes a ako print target mozes rovno pouzit `#a{{ action.pk }}`

Reply:
Nad týmto som premýšlal keď som to tvoril a nechal som tam zámerne obe premenné. Jedno je označenie akcie, druhé je označenie bloku, ktorý sa tlačí. Myslím, že to sú dve rozdielne veci, aj keď tu dosť súvisiace.

Ak by som ako print target použil `#a{{ action.pk }}` tak nikto nebude rozumieť tomu, prečo tam je nejaké `a`.

Vieš čo myslím?

Reply:
V takom pripade bude asi najpraktickejsie obalit cely box a akciou do dalsieho divu a tomu divu dat vlastne `print_id` vygenerovane cez `idgenerator`. Teda, ze potom ani nepouzivat to iste id na dve veci.

Ten novy div mozes dat bud z vonku okolo boxu, alebo z vnutra do boxu. Podla toho, ci chces aby sa pri tlaci vyrendroval aj ramcek okolo akcie.

Reply:
OK, asi ešte lepšie.